drivers/sx126x/sx126x_netdev.c

```diff
@@ -37,18 +40,8 @@
 const uint8_t llcc68_max_sf = LORA_SF11;
 const uint8_t sx126x_max_sf = LORA_SF12;
 
-#if IS_USED(MODULE_SX126X_STM32WL)
-static netdev_t *_dev;
-
-void isr_subghz_radio(void)
-{
-    /* Disable NVIC to avoid ISR conflict in CPU. */
-    NVIC_DisableIRQ(SUBGHZ_Radio_IRQn);
-    NVIC_ClearPendingIRQ(SUBGHZ_Radio_IRQn);
-    netdev_trigger_event_isr(_dev);
-    cortexm_isr_end();
-}
-#endif
```

Comment:
this should still be there, otherwise the IRQ offloading mechanism of `netdev` won't work.
Since the HAL and `netdev` implementation will never be compiled together, you can just leave this function there